src/main/kotlin/com/github/adeynack/jsoncmp/guiComparison/GuiComparison.kt

```diff
@@ -0,0 +1,21 @@
+package com.github.adeynack.jsoncmp.guiComparison
+
+import com.github.adeynack.jsoncmp.Args
+import com.github.adeynack.jsoncmp.ComparisonBase
+import javax.swing.SwingUtilities
+import javax.swing.WindowConstants.EXIT_ON_CLOSE
+
+class GuiComparison : ComparisonBase() {
+
+    override fun compare(arguments: Args) {
+        val actual = loadFileContent(arguments.actualFilePath, true, "actual")
+        val expected = loadFileContent(arguments.expectedFilePath, true, "expected")
+        SwingUtilities.invokeLater {
```

Comment:
First frame creation doesn't have to be invoked from EDT (if I remember correctly), does it?